### ISSUE TITLE
Webhook fails to start on Rancher local 1.22

### ISSUE THREAD

**Issue by kinarashah** [rancher/rancher#35495]
Bring up Rancher with local management cluster on k8s 1.22. Rancher webhook is in a crashloop, logs:
```
level=info msg="Rancher-webhook version dev (HEAD) is starting"
level=info msg="Listening on :9443"
level=info msg="Active TLS secret cattle-webhook-tls (ver=3381) (count 1): map[listener.cattle.io/cn-rancher-webhook.cattle-system.svc:rancher-webhook.cattle-system.svc listener.cattle.io/hash:7aaa4f3db00be92fc19d8a277022303e8c09763c46838b928cb6b5566a34ae31]"
level=fatal msg="no matches for kind \"CustomResourceDefinition\" in version \"apiextensions.k8s.io/v1beta1\""
```

**Comment by kinarashah** [comment]
https://github.com/rancher/rancher/issues/34054

**Comment by kinarashah** [comment]
https://github.com/rancher/rancher/pull/35357/commits/d5fccf9054393ad2e5564e7e18bb32a719dc8ee0

**Comment by kinarashah** [comment]
- Available to test with v2.6-head, make sure it's an image after this build https://drone-publish.rancher.io/rancher/rancher/6862. (Image ID: `ffa2be0ed804`, commit/version: `f487c8819`) 
- Make sure `webhook` deployment in local cluster is running with no errors and correct image - `rancher/rancher-webhook:v0.2.2-rc1`

**Comment by sowmyav27** [comment]
**On 2.6.3-rc1**

- Rancher webhook is seen deployed successfully. on a HA rancher with local cluster = v1.22.3-rancher1-1 using RKE 1.3.3-rc2
- app version: `rancher-webhook:1.0.2+up0.2.2-rc1`
- App imageApp image:

```
  repository: rancher/rancher-webhook
  tag: v0.2.2-rc2
```
